cfgov/v1/wagtail_hooks.py

```diff
@@ -365,21 +366,39 @@ def hide_snippets_menu_item(request, menu_items):
                      if item.url != reverse('wagtailsnippets:index')]
 
 
-# Override list of allowed tags in a RichTextField
-@hooks.register('construct_whitelister_element_rules')
-def whitelister_element_rules():
-    allow_html_class = attribute_rule({
-        'class': True,
-        'itemprop': True,
-        'itemscope': True,
-        'itemtype': True,
-    })
+if wagtail.VERSION <= (1, 14):
+    # Override list of allowed tags in a RichTextField
+    @hooks.register('construct_whitelister_element_rules')
+    def whitelister_element_rules():
+        allow_html_class = attribute_rule({
+            'class': True,
+            'itemprop': True,
+            'itemscope': True,
+            'itemtype': True,
+        })
 
-    allowed_tags = ['aside', 'h4', 'h3', 'p', 'span',
-                    'table', 'tr', 'th', 'td', 'tbody', 'thead', 'tfoot',
-                    'col', 'colgroup']
+        allowed_tags = ['aside', 'h4', 'h3', 'p', 'span',
+                        'table', 'tr', 'th', 'td', 'tbody', 'thead', 'tfoot',
+                        'col', 'colgroup']
 
-    return {tag: allow_html_class for tag in allowed_tags}
+        return {tag: allow_html_class for tag in allowed_tags}
+
+
+# construct_whitelister_element_rules are depricated in Wagtail 2.0
+# https://docs.wagtail.io/en/stable/releases/
+# 2.0.html#construct-whitelister-element-rules-hook-is-deprecated
+if wagtail.VERSION >= (2, 0):
+    @hooks.register('register_rich_text_features')
+    def blockquote_feature(features):
+
+        # register a feature 'blockquote'
+        # which whitelists the <blockquote> element
+        features.register_converter_rule('editorhtml', 'blockquote', [
+            WhitelistRule('blockquote', allow_without_attributes),
+        ])
+
+        # add 'blockquote' to the default feature set
+        features.default_features.append('blockquote')
```

Comment:
Per discussion in scrum just now, we realized that the `whitelister_element_rules` _only_ apply Hallo, and since we _probably_ don't want to allow any of these tags (pending investigation) within our lone remaining use of Hallo (table cells), we can probably wipe this whole section for Wagtail 2.

```suggestion
```